.github/workflows/release.yml

```diff
@@ -0,0 +1,30 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@master
+
+      - name: Setup Node.js 10.x
+        uses: actions/setup-node@master
+        with:
+          version: 10.x
+
+      - name: Install Bolt
+        run: npm install --global bolt
+
+      - name: Install Dependencies
+        run: bolt
+
+      - name: Create Release Pull Request
+        uses: changesets/action@master
```

Comment:
Is this a pre-canned action which already exists somewhere else?

Reply:
Yes, I wrote it, https://github.com/changesets/action

Reply:
Aha, Excellent!